blueprints/lcow.yml

```diff
@@ -1,10 +1,14 @@
 kernel:
-  image: linuxkit/kernel:4.12.14
+  # XXX Use the content hash as there are several 4.12.14 images on hub
+  image: linuxkit/kernel:4.12.14-c4d19a4de21def5a5134a6f70ef7212c31104bf1
```

Comment:
OK, but does a force push not overwrite?

Is it really possible to have multiple tags with the same name? (I suspect you meant something else)

Reply:
It's to prevent the case where you may have had a `linuxkit/kernel:4.12.14` in your local docker store so I'm explicitly pulling the version with the hash. I'll drop the hash again on 4.12.15 update, if there is one

Reply:
OIC, the unusual case here is that we are doing a rev of the packaging (and patches) but not the underlying kernel version so there is a greater than usual chance of picking up a stale image.

This seems like a shortcoming in our packaging strategy, but not one which should be addressed in this PR. So my LGTM stands once things go green.

Reply:
we do have `moby build -pull` which will always pull. Any downstream builds should use this. But me as a dev, I never do because I typically have most/all the images locally, so this is juts to protect me (and others).